Example/Example/ViewController.swift

```diff
@@ -5,52 +5,177 @@
+                    let formatter = NumberFormatter()
+                    formatter.locale = .current
+                    formatter.numberStyle = .scientific
+                    $0.formatter = formatter
+                }
+                <<< IntFloatLabelRow() {
+                    $0.title = "Spell out style"
+                    $0.value = 2017
+                    let formatter = NumberFormatter()
+                    formatter.locale = .current
+                    formatter.numberStyle = .spellOut
+                    $0.formatter = formatter
+                }
+                <<< DecimalFloatLabelRow() {
+                    $0.title = "Energy: Jules to calories"
+                    $0.value = 100.0
+                    let formatter = EnergyFormatter()
+                    $0.formatter = formatter
+                }
+                <<< IntFloatLabelRow() {
+                    $0.title = "Weight: Kg to lb"
+                    $0.value = 1000
+                    $0.formatter = MassFormatter()
+                }
+            
+            +++ Section(header: "Custom formatter", footer: "Custom formatter: CurrencyFormatter")
+                <<< DecimalFloatLabelRow() {
+                    $0.title = "Currency style"
+                    $0.value = 2000.00
+                    $0.useFormatterDuringInput = true
+                    let formatter = CurrencyFormatter()
+                    formatter.locale = .current
+                    formatter.numberStyle = .currency
+                    $0.formatter = formatter
+                }
     }
 
+    class CurrencyFormatter : NumberFormatter, FormatterProtocol {
+        
+        override func getObjectValue(_ obj: AutoreleasingUnsafeMutablePointer<AnyObject?>?, for string: String, range rangep: UnsafeMutablePointer<NSRange>?) throws {
+            guard obj != nil else { return }
+            let str = string.components(separatedBy: CharacterSet.decimalDigits.inverted).joined(separator: "")
+            obj?.pointee = NSNumber(value: (Double(str) ?? 0.0)/Double(pow(10.0, Double(minimumFractionDigits))))
+        }
+        
+        func getNewPosition(forPosition position: UITextPosition, inTextInput textInput: UITextInput, oldValue: String?, newValue: String?) -> UITextPosition {
+            return textInput.position(from: position, offset:((newValue?.characters.count ?? 0) - (oldValue?.characters.count ?? 0))) ?? position
+        }
+        
+    }
+
+}
+
+// MARK: Logo view
+class LogoViewNib: UIView {
+    
+    @IBOutlet weak var imageView: UIImageView!
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+}
+
+class LogoView: UIView {
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        let imageView = UIImageView(image: UIImage(named: "Xmartlabs"))
+        imageView.frame = CGRect(x: 0, y: 0, width: 318, height: 130)
+        imageView.autoresizingMask = .flexibleWidth
```

Comment:
if you have a nib for this view i don't think you really need to layout the `imageview` programmatically and you could just set up constraints using the interface builder